manifests/openshift-resource-topology-exporter-ds.yaml

```diff
@@ -55,14 +52,15 @@ spec:
           image: ${RTE_CONTAINER_IMAGE}
           command:
             - /bin/resource-topology-exporter
-            - --export-namespace=${RTE_NAMESPACE}
+            - -v=5
```

Comment:
are we ok with hardcoded value here?

Reply:
yes, because this manifest is used only in this repo CI. The recommended way to deploy is using the numaresources-operator, which must set the verbosity in a smarter way.